appveyor.yml

```diff
@@ -1,75 +1,84 @@
 environment:
 
-  CONDA_INSTALL_LOCN: "C:\\conda"
-
-  # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
-  # /E:ON and /V:ON options are not enabled in the batch script interpreter
-  # See: http://stackoverflow.com/a/13751649/163740
-  CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
-
   BINSTAR_TOKEN:
     # Generated with "anaconda auth --create --name conda-recipes-scitools-2016 -o SciTools" and
     # encrypted on ci.appveyor.com/tools/encrypt.
     secure: vEXaxIcVqImAv8mMqEGotj4Mym4uLVkHhf3jlksGimFJ88RcaF/hxdEUSt6HTxPr
 
   matrix:
-    # Note: Because we have to separate the py2 and py3 components due to compiler version, we have a race condition for non-python packages.
-    # Not sure how to resolve this, but maybe we should be tracking the VS version in the build string anyway?
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "27"
+    - TARGET_ARCH: x64
       PY_CONDITION: "python >=2.7,<3"
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "34"
+      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
+
+    - TARGET_ARCH: x64
       PY_CONDITION: "python >=3.4,<3.5"
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "35"
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
+
+    - TARGET_ARCH: x64
       PY_CONDITION: "python >=3.5,<3.6"
-    - TARGET_ARCH: "x64"
-      CONDA_PY: "27"
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
+
+    - TARGET_ARCH: x64
+      PY_CONDITION: "python >=3.6,<3.7"
+      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
+
+    - TARGET_ARCH: x86
       PY_CONDITION: "python >=2.7,<3"
-    - TARGET_ARCH: "x64"
-      CONDA_PY: "34"
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x86
       PY_CONDITION: "python >=3.4,<3.5"
-    - TARGET_ARCH: "x64"
-      CONDA_PY: "35"
+      CONDA_INSTALL_LOCN: C:\\Miniconda35
```

Comment:
I am "spelling out" the local installation of `miniconda` on `AppVeyor` to make the CI a little bit faster. But note that AppVeyor does not have `miniconda` for `Python 3.4`, so that will start with Python 3.5 and will be a little bit slower b/c `conda-build` will download `3.4` from there.